src/commcare_cloud/ansible/roles/commcarehq/templates/celery_task_kill.sh.j2

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
+
+# Process to soft and hard kill old celery jobs.
+#
+
+HOURS=$1
+MAX_MINUTES=`expr ${HOURS} \* 60 \* 60`
+
+SIG_NAME=$2
+
+PID=$(ps -eo comm,pid,etimes | awk '/celeryd:/ { if ( $4 > '"${MAX_MINUTES}"' ) { print $3 } }');
```

Comment:
I think $4 & $3 are incorrect (should be $3 and $2), also not 100% sure about the quoting around MAX_MINUTES

Reply:
MAX_Minutes is working as expected. Tested.

Reply:
I think if you test it out you'll find it's not working. The `ps` command only has 3 columns so $4 will be blank. $3 is the elapsed time and $2 is the PID:
```
~$ ps -eo comm,pid,etimes | head | ps -eo comm,pid,etimes | head | awk '{ print $2, $3 }'

PID ELAPSED
1 1372223
2 1372223
3 1372223
4 1372223
6 1372223
9 1372223
10 1372223
11 1372223
12 1372223
```

Reply:
@snopoke The command which I'm using is slightly different. The celeryd process has some spaces in the output therefore it giving me four slots. 
example1:
```
ps -eo comm,pid,etimes | grep celeryd
[celeryd: celer  1167      62
[celeryd: celer  1210      43
[celeryd: celer  1212      43
[celeryd: celer  1254      33
[celeryd: celer  1258      23
[celeryd: celer  1259      22
```
example2: 
```
ps -eo comm,pid,etimes | awk '/celeryd:/'
[celeryd: celer  1167      53
[celeryd: celer  1210      34
[celeryd: celer  1212      34
[celeryd: celer  1253      24
[celeryd: celer  1254      24
```
actual cmd with test minutes 12960: 
```
---  command has space [celeryd: celer ---
ps -eo comm,pid,etimes | awk '/celeryd:/ { if ( $4 > 12960 ) { print $1, $2, $3, $4 } }';
command, pid, etimes
[celeryd: celer 11972 126769
[celeryd: celer 13088 126565
[celeryd: celer 28246 35571
[celeryd: celer 28247 35571
[celeryd: celer 28249 35571
``` 
pid result: took from above " [celeryd: celer **28249** 35571 "
```
> ps -ef | grep 28259
cchq      1526 28259  0 05:55 ?        00:00:00 [celeryd: celery@<privateip>_saved_exports_queue_0.1602100766_timestamp:ForkPoolWorker-595]
pkasha    1610   840  0 05:56 pts/0    00:00:00 grep --color=auto 28259
cchq     28259 28241  1 Oct07 ?        00:09:40 [celeryd: celery@privateip_saved_exports_queue_0.1602100766_timestamp:MainProcess] -active- (-A corehq worker --hostname=privateip_saved_exports_queue_0.1602100766_timestamp --events --loglevel=INFO --without-gossip --queues=saved_exports_queue --autoscale=3,0 -Ofair --prefetch-multiplier=4)
```
```
pid result: took from above " [celeryd: celer 28249 **35571** "
> ps -ef | grep 35571
pkasha    1912   840  0 05:58 pts/0    00:00:00 grep --color=auto 35571
```

Reply:
:+1: thanks for the explanation